pyslackers_website/blog/migrations/0001_initial.py

```diff
@@ -2,9 +2,9 @@
 # Generated by Django 1.11.2 on 2017-07-03 22:39
 from __future__ import unicode_literals
 
+import django.db.models.deletion
```

Comment:
There’s no reason to change old migration files, they’re auto generated and not held to the standards of the application. Please revert anything on migrations files